src/Upgrade/Infrastructure/Configuration/ConfigurationProvider.php

```diff
@@ -90,6 +120,24 @@ public function getAccessToken(): string
     }
 
     /**
+     * Specification:
+     * - The method return link to your own code source provider system.
+     * - By default left it empty.
+     *
+     * @throw \Upgrade\Infrastructure\Exception\EnvironmentVariableIsNotDefinedException
+     *
+     * @return string
+     */
+    public function getSourceCodeProviderUrl(): string
+    {
+        return (string)getenv('SOURCE_CODE_PROVIDER_URL');
+    }
+
+    /**
+     * Specification:
+     * - The method return GitHub organization name for your project.
```

Comment:
- Defines organisation name for source provider.

Reply:
++